diff -up dzen2-0.8.5/config.mk.check_environment dzen2-0.8.5/config.mk
--- dzen2-0.8.5/config.mk.check_environment	2007-08-18 08:18:50.000000000 +0200
+++ dzen2-0.8.5/config.mk	2007-12-26 13:40:06.000000000 +0100
@@ -4,40 +4,41 @@ VERSION = 0.8.5
 # Customize below to fit your system
 
 # paths
-PREFIX = /usr/local
-MANPREFIX = ${PREFIX}/share/man
+PREFIX ?= /usr/local
+MANPREFIX ?= ${PREFIX}/share/man
 
 X11INC = /usr/X11R6/include
 X11LIB = /usr/X11R6/lib
 INCS = -I. -I/usr/include -I${X11INC}
 
 # Configure the features you want to be supported
-# Only one of the following options has to be uncommented, 
-# all others must be commented!
-#
-# Uncomment: Remove # from the beginning of respective lines
-# Comment  : Add # to the beginning of the respective lines
 
-# Option 1: No Xinerama no XPM
-LIBS = -L/usr/lib -lc -L${X11LIB} -lX11
-CFLAGS = -Os ${INCS} -DVERSION=\"${VERSION}\"
+# Remove the # from the next line to enable Xinerama support
+#DZEN_XINERAMA=1
+
+# Remove the # from the next line to enable Xpm support
+#DZEN_XPM=1
 
-# Option 2: No Xinerama with XPM
-#LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 -lXpm
-#CFLAGS = -Os ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XPM
-
-# Option 3: With Xinerama no XPM
-#LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 -lXinerama
-#CFLAGS = -Os ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XINERAMA
-
-# Option 4: With Xinerama and XPM
-#LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 -lXinerama -lXpm
-#CFLAGS = -Os ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XINERAMA -DDZEN_XPM
+# Uncomment: Remove # from the beginning of respective lines
+# Comment  : Add # to the beginning of the respective lines
 
 # END of feature configuration
 
+CFLAGS ?= -Os
+CFLAGS += ${INCS} -DVERSION=\"${VERSION}\"
+LIBS = -L/usr/lib -lc -L${X11LIB} -lX11
+
+ifeq ($(DZEN_XINERAMA),1)
+LIBS += -lXinerama
+CFLAGS += -DDZEN_XINERAMA
+endif
+
+ifeq ($(DZEN_XPM), 1)
+LIBS += -lXpm
+CFLAGS += -DDZEN_XPM
+endif
 
-LDFLAGS = ${LIBS}
+LDFLAGS = ${CFLAGS} ${LIBS}
 
 # Solaris, uncomment for Solaris
 #CFLAGS = -fast ${INCS} -DVERSION=\"${VERSION}\"
diff -up dzen2-0.8.5/gadgets/config.mk.check_environment dzen2-0.8.5/gadgets/config.mk
--- dzen2-0.8.5/gadgets/config.mk.check_environment	2007-08-18 08:18:50.000000000 +0200
+++ dzen2-0.8.5/gadgets/config.mk	2007-12-26 13:39:04.000000000 +0100
@@ -1,8 +1,8 @@
 # Customize below to fit your system
 
 # paths
-PREFIX = /usr/local
-MANPREFIX = ${PREFIX}/share/man
+PREFIX ?= /usr/local
+MANPREFIX ?= ${PREFIX}/share/man
 
 X11INC = /usr/X11R6/include
 INCS = -I. -I/usr/include -I${X11INC}
@@ -10,8 +10,9 @@ INCS = -I. -I/usr/include -I${X11INC}
 X11LIB = /usr/X11R6/lib
 LIBS = -L/usr/lib 
 
-CFLAGS = -Os ${INCS} 
-LDFLAGS = ${LIBS}
+CFLAGS ?= -Os 
+CFLAGS += ${INCS} 
+LDFLAGS = ${CFLAGS} ${LIBS}
 
 # compiler and linker
 CC = gcc
