diff -up dzen2-0.8.5-2010-01-04/config.mk.fedora-config dzen2-0.8.5-2010-01-04/config.mk
--- dzen2-0.8.5-2010-01-04/config.mk.fedora-config	2010-01-04 23:53:18.000000000 +0100
+++ dzen2-0.8.5-2010-01-04/config.mk	2010-01-04 23:53:25.000000000 +0100
@@ -39,8 +39,8 @@ INCS = -I. -I/usr/include -I${X11INC}
 
 
 ## Option 5: With XFT
-LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 `pkg-config --libs xft`
-CFLAGS = -Wall -Os ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XFT `pkg-config --cflags xft`
+#LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 `pkg-config --libs xft`
+#CFLAGS = -Wall -Os ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XFT `pkg-config --cflags xft`
 
 
 ## Option 6: With XPM and XFT
@@ -49,15 +49,15 @@ CFLAGS = -Wall -Os ${INCS} -DVERSION=\"$
 
 
 ## Option 7: With Xinerama and XPM and XFT
-#LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 -lXinerama -lXpm `pkg-config --libs xft`
-#CFLAGS = -Wall -Os ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XINERAMA -DDZEN_XPM -DDZEN_XFT `pkg-config --cflags xft`
+LIBS = -L$(LIBDIR) -lc -L${X11LIB} -lX11 -lXinerama -lXpm `pkg-config --libs xft`
+CFLAGS = $(RPM_OPT_FLAGS) -Wall ${INCS} -DVERSION=\"${VERSION}\" -DDZEN_XINERAMA -DDZEN_XPM -DDZEN_XFT `pkg-config --cflags xft`
 
 
 
 # END of feature configuration
 
 
-LDFLAGS = ${LIBS}
+LDFLAGS = $(RPM_OPT_FLAGS) ${LIBS}
 
 # Solaris, uncomment for Solaris
 #CFLAGS = -fast ${INCS} -DVERSION=\"${VERSION}\"
diff -up dzen2-0.8.5-2010-01-04/gadgets/config.mk.fedora-config dzen2-0.8.5-2010-01-04/gadgets/config.mk
--- dzen2-0.8.5-2010-01-04/gadgets/config.mk.fedora-config	2007-07-20 11:30:48.000000000 +0200
+++ dzen2-0.8.5-2010-01-04/gadgets/config.mk	2010-01-04 23:53:04.000000000 +0100
@@ -8,10 +8,10 @@ X11INC = /usr/X11R6/include
 INCS = -I. -I/usr/include -I${X11INC}
 
 X11LIB = /usr/X11R6/lib
-LIBS = -L/usr/lib 
+LIBS = -L$(LIBDIR)
 
-CFLAGS = -Os ${INCS} 
-LDFLAGS = ${LIBS}
+CFLAGS = $(RPM_OPT_FLAGS) -Os ${INCS} 
+LDFLAGS = $(RPM_OPT_FLAGS) ${LIBS}
 
 # compiler and linker
 CC = gcc
